### PR TITLE
When looking for romanized languages, ignore Romansh (rm)

### LINKED ISSUE #84: invalid city-name, i dont get it..
hi,
( this is most-likely not a bug, but a head-scratching case. i don't get it...)
( also, this points to the "production" version maps.wikimedia.org -- as used in wikidata.org ..  )

working on 
https://www.wikidata.org/wiki/Q183104
the geo shows this map
https://maps.wikimedia.org/img/osm-intl,13,47.238055555556,9.5983333333334,310x180.png?lang=en

The name of the city shows "**Sogn Pieder**", but should be "**Feldkirch**"

this seems to be the "räteromanisch" ( = a language) name of the city. ( https://de.wikipedia.org/wiki/Feldkirch )
This makes no sense. Räterromanisch is a small, valid language in Switzerland (nearby), but no official one in Austria.

For me this looks like: at as some-point-in-time this was as-is in the data.
With "current" i cant re-create this behavior. ( i can also switch the "?lang=en" to "de", "fr".. nothing changes )

Maybe some "very stable" server side caching issue ?

.ka7

### PR DESCRIPTION
**Pull request by mooeypoo**
Since our prefix is 'name_xx', the lookup for languages that end
with _rm can also pick up 'name_rm' which isn't a romanized script
but rather the language "Romansh".

This changes the lookup fallback script to take that into account.

Fixes kartotherian/kartotherian#84

### CODE REVIEW

File: test/langPickerTest.js [hunk 1]
```diff
@@ -110,6 +110,26 @@ describe('LanguagePicker: Pick the correct language', () => {
       ],
       expected: 'foo-Latn value',
     },
+    {
+      msg: 'Edge case: Values have value for "rm" which is also the suffix' +
```
Comment (mdholloway) [inline test/langPickerTest.js]: nit: could use a space at the end of this quoted text or the beginning in the next line

**Comment by mdholloway** [comment]
Thanks very much for the patch, @mooeypoo!  The code changes make sense and look good to me.  One nit inline, and also I'm not sure we want to include changes to package-lock.json as part of this commit. 

@mateusbs17, do you have a local environment with data for Austria (or somewhere Romansh labels exist) to test this in?  Alternatively, if you're planning on deploying to maps1004 soon, we could just merge and deploy and double-check this there afterward.

**Comment by thesocialdev** [comment]
@mdholloway I will set up the environment to test it and let you know the outcome!

File: test/langPickerTest.js (hunk 1 above)
Comment (mooeypoo) [reply test/langPickerTest.js]: hah, whoops :D added

**Comment by mooeypoo** [comment]
Okay, I didn't notice I included package-lock.json initially, so I took it out now, and I fixed the small spacing issue. This should be good to go, after testing. Let me know if there are any issues!

**Review by thesocialdev — APPROVED** [review]
